src/module/todolist.js

```diff
@@ -0,0 +1,51 @@
+import TodoInputSection from './TodoInputSection';
+import TodoListSection from './TodoListSection';
+import TodoStatusSection from './TodoStatusSection';
+import { CSS_CLASS, STRING_LITERAL } from './Constant';
+
+export default class TodoList {
+  constructor(containerElement, todos = []) {
+    this.itemsCount = {
+      total: 0,
+      completed: 0,
+    };
+
+    this.todoListContainer = containerElement;
+
+    let newTodoList = new TodoListSection(this.itemsCount, this.updateTodoStatus.bind(this));
+    this.todoListElement = newTodoList.todoListElement;
+
+    let newTodoStatus = new TodoStatusSection(newTodoList);
+    this.todoStatusElement = newTodoStatus.statusElement;
+    this.activeCounter = newTodoStatus.activeCounter;
+    this.clearCompletedBtn = newTodoStatus.clearCompletedBtn;
+
+    let todoInputSection = new TodoInputSection(newTodoList);
+    this.todoInputElement = todoInputSection.inputElement;
+
+    if (todos && todos.length >= 0) {
+      todos.forEach((todoItem) => newTodoList.addNewTodoItem(todoItem));
+    } else {
+      throw new TypeError('second parameter was must transferred Array type.'); // 파라미터 에러 체크
+    }
```

Comment:
예외 케이스를 먼저 쓰면 좋을 것 같네요!
```suggestion
    if (!(todos && todos.length >= 0)) {
      throw new TypeError('second parameter was must transferred Array type.'); // 파라미터 에러 체크
    }
    
    todos.forEach((todoItem) => newTodoList.addNewTodoItem(todoItem));      
```